src/styles/variables.scss

```
@@ -0,0 +1,7 @@
+@mixin margin_zero {
+  margin: 0;
+}
+
+$header_logo: 40px;
+$card-img: 160px;
```

Може так?
```suggestion
$logo-size: 40px;
$card-img-size: 160px;
```
В інших місцях `$logo-size` можна буде просто через обчислення змінити на більше чи менше або як доцільно буде